pkg/cmd/opts/gocloud_integration_test.go

```diff
@@ -0,0 +1,36 @@
+// +build integration
+
+package opts_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/jenkins-x/jx/pkg/cmd/testhelpers"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/jenkins-x/jx/pkg/cmd/clients"
+	"github.com/jenkins-x/jx/pkg/cmd/opts"
+	"github.com/jenkins-x/jx/pkg/log"
+)
+
+func TestCommonOptions_EnsureGCloudBinaryInstalled(t *testing.T) {
+	o := opts.NewCommonOptionsWithFactory(clients.NewFactory())
+	o.NoBrew = true
+
+	origJXHome, testJXHome, err := testhelpers.CreateTestJxHomeDir()
+	if err != nil {
+		t.Errorf("Failed to create a test JX Home directory")
```

Comment:
We can simply `assert.NoError(t, err, "Failed to create a test JX Home directory")`

Reply:
Thanks :)